cli/azd/extensions/azure.ai.agents/internal/cmd/init.go

```diff
@@ -1065,6 +1058,85 @@ func (a *InitAction) addToProject(ctx context.Context, targetDir string, agentMa
 	return nil
 }
 
+func (a *InitAction) populateContainerSettings(ctx context.Context) (*project.ContainerSettings, error) {
```

Comment:
In the use case where the user does not go through the `init` flow - are you still applying the defaults at deploy (agent creation) time?